source/reactor/src/EventReaction.h

```diff
@@ -0,0 +1,86 @@
+#ifndef Reactor_EventReaction_H
+#define Reactor_EventReaction_H
+
+#include "reactor/EventHandlers.h"
+
+#include <time.h>
+
+namespace u {
+namespace reactor {
+
+class EventReaction {
+public:
+    /**
+     * Class constructor.
+     *
+     * @param fd File descriptor.
+     * @param timeout Timeout value (in milliseconds).
+     * @param handlers Set of event handlers.
+     */
+    EventReaction(int fd, long timeout, const EventHandlers &handlers);
+
+    /**
+     * Returns file descriptor.
+     * @return File descriptor.
+     */
+    int getFd(void) const;
```

Comment:
Coding style:
```suggestion
    int fd(void) const;
```
(I don't think we say in the coding style to remove the get part, but we should :-)

Same comment in other places as well.

Also, I prefer to have these simple functions to be defined inline.